### DISCUSSION TITLE
Secret Decks Meeting  -  25th May 2021

### DISCUSSION THREAD

**Discussion by stephen-rowan** [SecretDecks/Documentation#28] [Meetings]
# Secret Decks Meetings
25th May 2021

## Attendees
Heather, Juliette, Gianluca, Dan, Stephen W. (remote), Tyler W. (remote) Alberto (remote)

## Agenda
- [x] discuss mindfulness policy - **will be discussed within github!**
- [x] finish writing newsletter! - **will be done by @evolayak and @DanM3rcurius by end of the week!**
- [x]  Github [Workflow](https://github.com/SecretDecks/Documentation/discussions/4) - **Discussion ongoing in Thread!**
- [x] [Contribution Survey](https://github.com/NFT-DAO/Governance-HOLON/discussions/96) - **adapt and implement!**
- [x] [Introduction Thread](https://github.com/SecretDecks/Documentation/discussions/2) - **Ongoing!**

## Discussion
- introduction of remote members (Alberto M., Stephen & Tyler)
- how to continue with dev?
- @bUxEE : create library for node CLI; 
- @DanM3rcurius : Explain product roadmap and technical stack 
- Townhall Tuesday works for everyone!
- Mindfulness, Contributions and Code of Conduct Policies unilateral decision to commence with those
- @SecretDecks/founding-core addition to introduction texts: -> what one can contribute, add schedule (timezone)
- Think about fund (Catalyst) application when doing contribution survey & policy!

### Contribution 
- how to contribute (how to track) , code of conduct: be respectful (Know & Read Thy Self)
- How to contribute (part of onboarding) , start with github. Keep flow in place! Propose tools to use! Never pressure people, let them develop in their own way. Nurture not manage!
- **Let it happen as it is now.**
- Talk about the website compendium ( how it can be a primer for the contribution policy)

**Comment by DanM3rcurius** [comment]
@SecretDecks/founding-core This is how we'll keep track of meeting minutes. As soon as someone has a topic to discuss, one can create a new discussion in the meeting **[category](https://github.com/SecretDecks/Documentation/discussions/categories/meetings)**

**Reply by stephen-rowan** [reply]
btw - anyone can pin or unpin a discussion on the main discussion page (https://github.com/SecretDecks/Documentation/discussions). To do this select from the right hand menu on a discussion thread. I pinned the [Welcome and Introduction Thread](https://github.com/SecretDecks/Documentation/discussions/2) and this meeting as examples.
